content/topics/intro-to-typescript/_index.md

```diff
@@ -5,11 +5,28 @@ ready: true
 title: Introduction to Typescript
 ---
 
+### What is Typescript
+
+- TypeScript is an extension of JavaScript that adds types as well as other language features.
+- Typescript was created to make it easier to write code that is compatible with the JavaScript language.
+- As its name implies, adding types to JavaScript is the primary purpose of TypeScript, and it is the reason that people use it today.
```

Comment:
types is an optional in TypeScript, there are other significant features you may perhaps want to highlight, like

strong static typing, compilation, and object-oriented programming